client/extensions/woocommerce/app/products/product-variation-types-form.js

```diff
@@ -75,20 +75,21 @@ export default class ProductVariationTypesForm extends Component {
 	}
 
 	renderInputs( attribute, index ) {
+		const { translate } = this.props;
 		const { attributeNames } = this.state;
 		const attributeName = attributeNames && attributeNames[ attribute.uid ] || attribute.name;
 		return (
 			<div key={ index } className="products__variation-types-form-fieldset">
 				<FormTextInput
-					placeholder={ i18n.translate( 'Color' ) }
+					placeholder={ translate( 'Color' ) }
 					value={ attributeName }
 					id={ attribute.uid }
 					name="type"
 					className="products__variation-types-form-field"
 					onChange={ this.updateNameHandler }
 				/>
 				<TokenField
-					placeholder={ i18n.translate( 'Red, Green, Blue' ) }
+					placeholder={ translate( 'Red, Green, Blue' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Blue & Red' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).